source/ui/REIXS/REIXSSampleManagementPre2013Widget.h

```diff
@@ -0,0 +1,89 @@
+along with Acquaman.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef REIXSSAMPLEPOSITIONPRE2013VIEW_H
+#define REIXSSAMPLEPOSITIONPRE2013VIEW_H
+
+#include <QWidget>
+
+class QGridLayout;
+class QUrl;
+class QGroupBox;
+
+class AMTopFrame2;
+
+
+#include "ui/dataman/AMSamplePlatePre2013View.h"
+class AMSamplePlatePre2013View;
+class AMSampleManipulator;
+
+#ifdef AM_MOBILITY_VIDEO_ENABLED
+class AMBeamlineCameraWidget;
+#endif
+
+/// This widget provides a complete full-screen view for users to view, move, align, and tag samples in the machine. It simply combines three widgets: an AMCrosshairVideoWidget for the video, whatever widget you want to provide for controlling the sample manipulator, and an AMSamplePlateView. We create the video widget and sample plate view for you.
+class REIXSSampleManagementPre2013Widget : public QWidget
+{
+Q_OBJECT
+public:
+	/// Construct the widget.
+	/*!
+\param manipulatorWidget Any kind of widget you want to include below the video in the lower-left (usually, for directly controlling the sample manipulator.)
+\param sampleCameraUrl1 The url of the video feed to open. (You can add more video feeds later by accessing the cameraWidget() and calling addSource().)
+\param sampleCameraDescription The title the user will see for the camera view (ex: "Endstation Top View").
+\param samplePlate A pointer that will be passed to the AMSamplePlateView constructor as the sample plate to work with; it can be 0 if the plate view should create its own plate.
+\param manipulator [Optional] If you want the AMSamplePlateView to be able to mark and move sample positions, you should include an AMSampleManipulator instance \c manipulator, which will be passed to the AMSamplePlateView.*/
+	REIXSSampleManagementPre2013Widget(QWidget *manipulatorWidget, const QUrl& sampleCameraUrl1, const QUrl& sampleCameraUrl2, AMSamplePlatePre2013* samplePlate = 0, AMSampleManipulator* manipulator = 0, QWidget *parent = 0);
+	virtual ~REIXSSampleManagementPre2013Widget();
+
+	/// Accesses the sample plate view
+	AMSamplePlatePre2013View* samplePlateView() { return plateView_; }
+#ifdef AM_MOBILITY_VIDEO_ENABLED
+	/// Access the camera widget.
+	AMBeamlineCameraWidget* cameraWidget() { return cameraWidget1_; }
+	AMBeamlineCameraWidget* cameraWidget2() { return cameraWidget2_; }
+#endif
+	/// Access the user-suppplied manipulator widget
+	QWidget* manipulatorWidget() { return manipulatorWidget_; }
+signals:
+	/// Forwards the newSamplePlateSelected() from the AMSamplePlateView.
+	void newSamplePlateSelected(AMSamplePlatePre2013 *selectedPlate);
+
+protected slots:
+	/// responds to and forwards the newSamplePlateSelected() signal from AMSamplePlate.
+	void onNewSamplePlateSelected();
+
+protected:
+
+#ifdef AM_MOBILITY_VIDEO_ENABLED
+	AMBeamlineCameraWidget *cameraWidget1_;
+	AMBeamlineCameraWidget *cameraWidget2_;
+#endif
+
+	AMSamplePlatePre2013View *plateView_;
+	QWidget *manipulatorWidget_;
+
+	AMTopFrame2 *topFrame_;
```

Comment:
The main reason is so that you can make use of the convenience methods from `AMMainWindow` and continue to have the same look and feel and then we can remove the class since it is mostly an unnecessary class that isn't used anywhere else.